docs/current_docs/cookbook/snippets/builds/dockerfile-context/typescript/index.ts

```diff
@@ -1,4 +1,4 @@
-import { dag, Directory, File, object, func } from "@dagger.io/dagger"
+import { dag, Directory, File, object, func } from "@dagger.io/dagger";
```

Comment:
nit/question(non blocking): do we care about `;` at the end of typescript lines?